getting-started/templates/systemlink-values.yaml

```diff
@@ -570,6 +592,10 @@ dataframeservice:
     # <ATTENTION> This must be overridden if not using the SLE MinIO instance.
     ##
     port: *minioPort
+    ## Maximum number of concurrent connections to S3.
```

Comment:
We may want to add a clarifying comment that this is connections per replica.

Reply:
Added "per replica" to the comment